### ISSUE TITLE
Accessibility - Incorrect keyboard navigation in iOS app when tabs are on the bottom

### ISSUE THREAD

**Issue by elless** [ionic-team/ionic#13214]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**
In iOS: When using an external keyboard to navigate in an Ionic app with tabs on the bottom -  the navigation starts at the bottom of the page on the tabs, and only after goes back to the top page to go over the page in order.

**Expected behavior:**
If tabs are on the bottom - they should be called out last.

**Steps to reproduce:**
1. Create simple Ionic app with tabs
2. Connect external keyboard
3. Turn on voice over/voice assistant
4. Navigate over the elements using the keyboard
Result: navigation starts on tabs
Expected: Navigation should end on tabs

**Other information:**

**Ionic info:**
```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.5.3
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 2.0.2
ios-deploy version: 1.9.2 
ios-sim version: 6.0.0 
OS: macOS Sierra
Node Version: v6.11.1
Xcode version: Xcode 9.0.1 Build version 9A1004
```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!
